bin/oneoff/remove_password_for_code_studio_admin.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+# Remove passwords for code studio admins
+
+batch_number = 0
+User.where(admin: true).where.not(encrypted_password: nil).find_in_batches(batch_size: 10) do |batch|
+  puts "PROCESSING: #{batch_number}..."
+  ActiveRecord::Base.transaction do
+    batch.each do |user|
+      user.update(encrypted_password: nil)
```

Comment:
recommend we use

```suggestion
      user.update!(encrypted_password: nil)
```

and catch and log errors